src/esp/bindings/AttributesBindings.cpp

```diff
@@ -234,6 +234,18 @@ void initAttributesBindings(py::module& m) {
       .value("TREE_TRAVERSAL",
              metadata::attributes::ArticulatedObjectLinkOrder::TreeTraversal);
 
+  //
+  // ==== AssetType ====
+  // Describes the type of asset used for rendering, collsions, or semantics
+  py::enum_<metadata::attributes::AssetType>(m, "AssetType")
```

Comment:
The getters provide this value as an enum. This does remind me, though , that this needs to be read only now, since we don't want manual manipulation of this as it could break things in arcane ways. The whole asset-type component system needs to be removed, since it has been obviated by various config settings.